frontend/src/components/vue_canvas/drawable_canvas.vue

```diff
@@ -218,7 +218,12 @@
         this.mouse_position,
         this.canvas_translate,
         this.canvas_element,
-        this.canvas_scale_global
+        this.canvas_scale_global,
+        this.instance_list,
+        this.canvas_height,
+        null,
```

Comment:
This is for previews
In this case not caring about zooms
Expectation is that it will then use full instance list